pdoc/theme.css

```diff
@@ -0,0 +1,30 @@
+:root {
+  --ctp-text: #4c4f69;
+  --ctp-subtext0: #6c6f85;
+  --ctp-base: #eff1f5;
+  --ctp-mantle: #e6e9ef;
+  --ctp-crust: #dce0e8;
+  --ctp-mauve: #8839ef;
+  --ctp-yellow: #df8e1d;
+  --ctp-sky: #04a5e5;
+  --ctp-blue: #1e66f5;
```

Comment:
![image](https://github.com/user-attachments/assets/db95a4ab-fb3e-4fe0-9786-f445c19947a9)

Reply:
yeah... got fed up of waiting literally 90 seconds for unpkg to respond on each refresh

Reply:
fixed with jsdelivr 👍